apps/web/playwright/lib/testUtils.ts

```diff
@@ -521,6 +521,14 @@ export async function confirmBooking(page: Page, url = "/api/book/event") {
   });
 }
 
+export async function gotoAndWaitForIdle(page: Page, url: string) {
```

Comment:
Is this the only change in this PR? Why is this needed?

I notice a lot of the E2E tests are timing out now that weren't before. I'm cautious about increase the timeout limit of our tests if we don't have to.